drv/lpc55-gpio-api/src/lib.rs

```diff
@@ -145,9 +145,13 @@ pub enum Digimode {
     Digital = 1,
 }
 
-impl Into<bool> for Digimode {
-    fn into(self) -> bool {
-        self == Self::Digital
+impl From<bool> for Digimode {
```

Comment:
I'm a little confused about this change - this removed the ability to convert `Digimode -> bool` and added the ability to convert `bool -> Digimode`. Was clippy complaining about implementing `Into` instead of `From`? If so, should this be `impl From<Digimode> for bool` to preserve the existing functionality?

Reply:
Turns out this code is actually unused so I removed it

Reply:
I have some changes in this area that do impl From here.
I need to put that in a separate PR.